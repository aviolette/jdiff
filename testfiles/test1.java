package foo.bar;

import bar.*;

public abstract class Foo
extends Bar
implements Foobar, BarFoo
{
    
    int bar = 1;
    public String foo = "foo";
    
    public int b()
    {
	java.lang.b();
	return 1;
    }

    public void foo()
	throws FooException, BarException
    {
	int bar = 0;
	int foob = this.bar;
	int methtest = java.lang.b(bar,foob);
	break bar;
    }
    

}
