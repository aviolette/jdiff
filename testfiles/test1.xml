<?xml version="1.0"?>

<!DOCTYPE tree_delta PUBLIC '-//DePaul//DTD Tree Delta//EN' 'file:///c:/my documents/classes/thesis/code/treedelta.dtd'>

<tree_delta>
  <update id="0">
    <value name="Package" isnull="true" value="" />
  </update>
  <delete id="1" />
  <delete id="46" />
</tree_delta>

