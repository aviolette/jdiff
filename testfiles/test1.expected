<?xml version="1.0"?>

<!DOCTYPE tree_delta PUBLIC '-//DePaul//DTD Tree Delta//EN' 'file:///c:/my documents/classes/thesis/code/treedelta.dtd'>

<tree_delta>
  <insert id="44" target="0" type="import" col="0" pos="1">
    <value name="OnDemand" value="true" />
    <value name="Name" value="foo" />
  </insert>
  <insert id="45" target="0" type="class" col="1" pos="1">
    <value name="Visibility" value="public" />
    <value name="Extends" value="Bar" />
    <value name="Static" value="false" />
    <value name="Final" value="false" />
    <value name="Name" value="Foo" />
    <value name="Abstract" value="false" />
  </insert>
  <move id="6" parent="45" col="0" pos="0" />
  <move id="11" parent="45" col="0" pos="1" />
  <move id="17" parent="45" col="0" pos="2" />
  <insert id="46" target="45" type="method" col="0" pos="3">
    <value name="Visibility" value="public" />
    <value name="Synchronized" value="false" />
    <value name="Native" value="false" />
    <value name="Static" value="false" />
    <value name="Final" value="false" />
    <value name="Name" value="foobar" />
    <value name="Abstract" value="false" />
  </insert>
  <move id="40" parent="45" col="0" pos="4" />
  <insert id="47" target="45" type="name" col="1" pos="0">
    <value name="Value" value="Foobar" />
  </insert>
  <insert id="48" target="45" type="name" col="1" pos="1">
    <value name="Value" value="BarFoo" />
  </insert>
  <insert id="49" target="46" type="block" col="1" pos="0" />
  <insert id="50" target="46" type="type" col="3" pos="1">
    <value name="Dimension" value="0" />
    <value name="Name" value="String" />
  </insert>
  <insert id="51" target="49" type="variable" col="0" pos="0">
    <value name="AsStatement" value="true" />
    <value name="Final" value="false" />
  </insert>
  <insert id="52" target="49" type="return" col="0" pos="1" />
  <insert id="53" target="39" type="break" col="0" pos="4">
    <value name="BreakLabel" value="alabel" />
  </insert>
  <insert id="54" target="51" type="type" col="0" pos="0">
    <value name="Dimension" value="0" />
    <value name="Name" value="String" />
  </insert>
  <insert id="55" target="51" type="vardec" col="1" pos="0">
    <value name="Dimension" value="0" />
    <value name="Name" value="s" />
  </insert>
  <insert id="56" target="52" type="name" col="0" pos="0">
    <value name="Value" value="s" />
  </insert>
  <insert id="57" target="55" type="literal" col="0" pos="0">
    <value name="Value" value="" />
  </insert>
  <insert id="58" target="57" type="type" col="0" pos="0">
    <value name="Dimension" value="0" />
    <value name="Name" value="String" />
  </insert>
  <insert id="59" target="35" type="name" col="1" pos="1">
    <value name="Value" value="java.lang.b" />
  </insert>
  <delete id="15" />
  <delete id="43" />
  <delete id="24" />
  <delete id="14" />
  <delete id="23" />
  <delete id="25" />
  <delete id="13" />
  <delete id="21" />
  <delete id="22" />
  <delete id="19" />
  <delete id="20" />
  <delete id="32" />
  <delete id="33" />
  <delete id="34" />
  <delete id="41" />
  <delete id="42" />
  <delete id="38" />
</tree_delta>

